### DISCUSSION TITLE
Keep hit objects horizontal when window is vertical in osu!standard

### DISCUSSION THREAD

**Discussion by smileyhead** [ppy/osu#13099] [Ideas]
**Describe the new feature:** The hit objects could stay in landscape even if the game window is in portrait.

**Proposal designs of the feature:** 

![image](https://user-images.githubusercontent.com/15932372/71125353-0dbeee00-21e7-11ea-95df-e0268545d375.png)
![image](https://user-images.githubusercontent.com/15932372/71125357-10214800-21e7-11ea-9763-f359e5da948b.png)

**Comment by bdach** [comment]
I'd rather advocate for blocking screen rotation entirely during gameplay myself.

**Comment by smileyhead** [comment]
Oh, I mainly meant this feature to help playing when portrait, not necessarily as a behaviour for mid-game rotation.

**Comment by bdach** [comment]
What I meant is that entering gameplay should force landscape. See several existing video apps (YouTube among others) for a precedent for this sort of behaviour.

**Comment by andy840119** [comment]
What does it display while changing windows size during gameplay or replay?

**Comment by smileyhead** [comment]
> What I meant is that entering gameplay should force landscape. See several existing video apps (YouTube among others) for a precedent for this sort of behaviour.

I'd argue properly supporting portrait mode would be preferable, since in landscape you have to stretch your fingers quite a bit, especially on taller phones.

> What does it display while changing windows size during gameplay or replay?

Changing window size during gameplay should be locked, unless the game is paused.
